src/dune_pkg/lock_dir.mli

```diff
@@ -31,6 +31,7 @@ module Pkg : sig
 
   val equal : t -> t -> bool
   val decode : (lock_dir:Path.Source.t -> Package_name.t -> t) Decoder.t
+  val files_dir : Package_name.t -> Path.Source.t -> Path.Source.t
```

Comment:
This needs a comment or a label on the `Path.Source.t` arg. Otherwise it's not clear what to pass as the `Path.Source.t` arg.